### PR TITLE
(SIMP-6639) Use newest rpms

### PR DESCRIPTION
**Pull request by m-morrone**
Update hirs_provisioner module to use newest rpms for testing and installation

SIMP-6639 #close

### CODE REVIEW

File: spec/acceptance/suites/default/05_install_tpms_spec.rb [hunk 1]
```diff
@@ -7,16 +7,16 @@
   def download_rpm_tarball_on(hirs_host, rpm_staging_dir)
     if hirs_host.host_hash[:roles].include?('tpm_2_0')
       tpm_rpms_tarball_url_string = ENV['BEAKER_tpm_2_0_rpms_tarball_url'] || \
-      'https://github.com/op-ct/simp-tpm2-rpms/releases/download/0.1.0-rpms/simp-tpm2-simulator-1119.0.0-0.el7.centos.x86_64.rpm'
+      'https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/simp-tpm2-simulator-1332.0.0-0.el7.x86_64.rpm'
     else 
       os = fact_on(hirs_host,'operatingsystemmajrelease')
       tpm_rpms_tarball_url_string = ENV['BEAKER_tpm_1_2_rpms_tarball_url'] || \
-      "https://github.com/m-morrone/simp-tpm12-rpms/releases/download/v0.1-beta/simp-tpm12-simulator-4769.0.0-0.el#{os}.x86_64.rpm"
+      "https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/#{os}/simp-tpm12-simulator-4769.0.0-0.el#{os}.x86_64.rpm"
```
Comment (trevor-vaughan) [inline spec/acceptance/suites/default/05_install_tpms_spec.rb]: Once the packages are published, we should get rid of this and just throw the repos into the nodesets and use `install_package_on`.

Comment (m-morrone) [reply spec/acceptance/suites/default/05_install_tpms_spec.rb]: Updated tests to pull rpms from packagecloud.

File: data/common.yaml [hunk 2]
```diff
@@ -1,26 +1,8 @@
 ---
 # hirs_provisioner::parameter_name: 'value'
```
Comment (lnemsick-simp) [inline data/common.yaml]: please add a changelog entry

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,10 @@
+* Fri Jun 14 2019 Michael Morrone <michael.morrone@onyxpoint.com> - 0.1.2
```
Comment (lnemsick-simp) [inline CHANGELOG]: - No version bump required
- Please combine into one 0.1.1 changelog
- Please put details of interest to end-user first

**Comment by m-morrone** [comment]
Link to gitlab pipeline for acceptance tests
https://gitlab.com/simp/pupmod-simp-hirs_provisioner/pipelines

**Comment by trevor-vaughan** [comment]
:hourglass_flowing_sand: https://gitlab.com/simp/pupmod-simp-hirs_provisioner/pipelines/66757305

**Pull request merged**